src/Symfony/Bundle/TwigBundle/Node/RenderNode.php

```diff
@@ -15,6 +15,8 @@
  * Represents a render node.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @deprecated Deprecated in 2.2, to be removed in 3.0.
```

Comment:
In other PR, we tried to normalize the message like this:
`@deprecated since version 2.2, to be removed in 3.0.`